qlib/workflow/expm.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, uri: Text, default_exp_name: Optional[Text]):
         self._active_exp_uri = None  # No active experiments. So it is set to None
         self._default_exp_name = default_exp_name
         self.active_experiment = None  # only one experiment can be active each time
-        logger.info(f"experiment manager uri is at {self.uri}")
+        logger.debug(f"experiment manager uri is at {self.uri}")
```

Comment:
Just copy below simple code into a py file and run
```python
import qlib
from qlib.contrib.data.handler import Alpha158


market = "csi300"
benchmark = "SH000300"
provider_uri = "<change to yours>"


if __name__ == "__main__":
    qlib.init(provider_uri=provider_uri, region='cn')
    pre_handler = Alpha158(
        instruments=market,
        start_time="2016-01-01",
        end_time="2022-10-31",
        infer_processors=[{"class": 'Fillna', "kwargs": {"fields_group": "feature"}}],
        learn_processors=[],
    )
```
you can see that line of log prints many many times. What I see in my run is
```
[17332:MainThread](2022-11-22 19:28:24,999) INFO - qlib.Initialization - [config.py:416] - default_conf: client.
[17332:MainThread](2022-11-22 19:28:25,634) INFO - qlib.workflow - [expm.py:40] - experiment manager uri is at file:D:\Documents\TradeResearch\qlib\debug_test\mlruns
[17332:MainThread](2022-11-22 19:28:25,634) INFO - qlib.Initialization - [__init__.py:74] - qlib successfully initialized based on client settings.
[17332:MainThread](2022-11-22 19:28:25,634) INFO - qlib.Initialization - [__init__.py:76] - data_path={'__DEFAULT_FREQ': WindowsPath('D:/Documents/TradeResearch/qlib_test/rqdata')}
[29836:MainThread](2022-11-22 19:28:32,145) INFO - qlib.workflow - [expm.py:40] - experiment manager uri is at file:D:\Documents\TradeResearch\qlib\debug_test\mlruns
[21576:MainThread](2022-11-22 19:28:32,190) INFO - qlib.workflow - [expm.py:40] - experiment manager uri is at file:D:\Documents\TradeResearch\qlib\debug_test\mlruns

+ another 46 repeats
```

Set up a breakpoint with an IDE, I see feature calculation of each instrument in `inst_calculator` registers in config and thus generates an exp-manager. Since feature calculation is paralleled, so many, many exp managers are created and that line of log gets printed.